copyright

```diff
@@ -1432,9 +1432,20 @@ License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (under the same license).
 
 Files:
- sounds/remnant afterburner.wav
-License: public-domain
-Comment: Based on public domain sounds taken from freesound.org, edit done by Saugia.
+ images/outfit/firelight*
```

Comment:
```suggestion
 images/icon/firelight*
 images/outfit/firelight*
```

Reply:
Looks like you did this in a copyright file that isn't updated- I'll commit the change real quick.